test/third_party_tests/test_native_executor.py

```diff
@@ -186,10 +186,6 @@ def test_should_run_query_in_mysql(self):
         self._execute_native_query()
         self._execute_evadb_query()
 
-        # Test error.
-        self._raise_error_on_multiple_creation()
```

Comment:
Why are we removing it?

Reply:
This test is actually invariant on which database engine we use. I currently just test on `Postgres`, so we should skip here.